test/pkg/uvm/lcow.go

```diff
@@ -69,8 +69,8 @@ func CreateAndStartLCOWFromOpts(ctx context.Context, tb testing.TB, opts *uvm.Op
 	}
 
 	vm, cleanup := CreateLCOW(ctx, tb, opts)
-	Start(ctx, tb, vm)
 	tb.Cleanup(func() { cleanup(ctx) })
+	Start(ctx, tb, vm)
```

Comment:
what's the issue here?

Reply:
if `Start` fails, it wont get to the cleanup call afterwards, so the uVM wont be closed and removed
its not a big issue, but still notcorrect